lib/internal/bootstrap/pre_execution.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const minimist = require('minimist')
+
+const argv = minimist(process.argv.slice(2), {
+    boolean: ['test', 'test-only']
```

Comment:
I think this module shouldn't use the `--test` flag

Reply:
I think it should accept it (so it's as close to the Node.js implementation as possible), and default to `true` so it still works if you omit it.

Reply:
sounds good. Or we modify `argv` retrieval logic to always include `--test`?

Reply:
OK so now the package exposes two binaries: `node-core-test` that does not assume `--test` (same as node), and `node--test` that defaults to `--test`. wdyt?